src/py/flwr/simulation/run_simulation.py

```diff
@@ -419,20 +550,29 @@ def _parse_args_run_simulation() -> argparse.ArgumentParser:
     )
     parser.add_argument(
         "--server-app",
-        required=True,
         help="For example: `server:app` or `project.package.module:wrapper.app`",
     )
     parser.add_argument(
         "--client-app",
-        required=True,
         help="For example: `client:app` or `project.package.module:wrapper.app`",
     )
     parser.add_argument(
         "--num-supernodes",
         type=int,
-        required=True,
         help="Number of simulated SuperNodes.",
     )
+    parser.add_argument(
+        "--app",
```

Comment:
Wdyt is better, `--app` or `--app-dir`?

Reply:
there is an `--app-dir` already present. I need to think what's the best way to have just one (instead of `--app` and `--app-dir`.